shared/src/Datadog.Trace.ClrProfiler.Native/process_helper.cpp

```diff
@@ -57,26 +86,35 @@ bool ProcessHelper::RunProcess(const std::string& processPath,
     // Because the equal sign is a separator, it must not be used in the name of an environment variable.
 
     PROCESS_INFORMATION pi;
-    if (CreateProcess(
+    if (!CreateProcess(
         nullptr,                  // lpApplicationName
         commandLine.data(),       // lpCommandLine
         nullptr,                  // lpProcessAttributes
         nullptr,                  // lpThreadAttributes
-        FALSE,                    // bInheritHandles
+        TRUE,                     // bInheritHandles
```

Comment:
I totally don't understand the meaning of `Ensure the write handle to the pipe for STDIN is not inherited.` followed by `bInheritHandles=TRUE` 😅

Reply:
`hStdinRead` must be inherited (so the child process can read from it), `hStdinWrite` mustn't (otherwise it stays open even after the parent closes it)